lib/typescript/httpclient/payload/ListTemplatesRequestPayload.ts

```diff
@@ -0,0 +1,4 @@
+export interface ListTemplatesRequestPayload {
+  name?: string;
+  sourceType: string;
+}
```

Comment:
newest change on _templates.list___: the sourceType is required so that only templates from the source will be fetched